go.mod

```diff
@@ -5,8 +5,9 @@ go 1.21
 require (
 	github.com/flyteorg/flyte/datacatalog v0.0.0-00010101000000-000000000000
 	github.com/flyteorg/flyte/flyteadmin v0.0.0-00010101000000-000000000000
-	github.com/flyteorg/flyte/flytepropeller v0.0.0-00010101000000-000000000000
-	github.com/flyteorg/flyte/flytestdlib v0.0.0-00010101000000-000000000000
+	github.com/flyteorg/flyte/flytepropeller v1.9.12
+	github.com/flyteorg/flyte/flytestdlib v1.9.12
+	github.com/flyteorg/flytectl v0.8.15
```

Comment:
We shouldn't update this.

Reply:
Just to clarify, leaving the [pseudo-versions](https://go.dev/ref/mod#pseudo-versions) help in show that those are replaced at the [end of this file](https://github.com/flyteorg/flyte/blob/master/go.mod#L234-L240).

Reply:
@eapolinario But I run `make go-tidy` on master branch and these lines of pseudo-versions changed. I do not change them manually. Do you know what happened? Is this a special golang behaviour?

Reply:
![image](https://github.com/flyteorg/flyte/assets/47914085/a61b19dd-4fde-44ee-b800-8fde38bd6df3)

Reply:
By the way, @pingsutw said that he also ran `make go-tidy` on the latest commit on the master branch but those pseudo-versions do not change.